packages/reporter/cypress/integration/aliases_spec.coffee

```diff
@@ -216,7 +216,7 @@ describe "aliases", ->
           .within ->
             cy.contains('.command-alias-count', '1')
             cy.contains('.command-alias', '@getPosts')
-              .trigger("mouseover")
+              .trigger("mouseover", { force: true })
```

Comment:
ℹ️ all tests weren't running locally, I've managed to fix this one that way, but I think it's not relevant to the current PR